Makefile

```diff
@@ -26,6 +26,10 @@ INSTALL_SIZE ?= $(shell du out/minikube-windows-amd64.exe | cut -f1)
 BUILDROOT_BRANCH ?= 2018.05.3
 REGISTRY?=gcr.io/k8s-minikube
 
+# Get git commit id
+COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
```

Comment:
let's limit it to first 6 characters of the commit . the full commit ID visually is too long.